libraries/AP_HAL/Device.h

```diff
@@ -26,13 +26,24 @@
  */
 class AP_HAL::Device {
 public:
+    enum BusType {
+        I2C,
+        SPI,
```

Comment:
`if (_dev->bus_type == AP_HAL::Device::SPI)` sounds pretty clear to me :-) hehe

It's already applied anyway.